hack/gen-swagger-doc/gen-swagger-docs.sh

```diff
@@ -84,7 +84,16 @@ if [ "$OUTPUT_FORMAT" = "html" ]; then
         "$WORKDIR/overview.adoc"
 
     # Generate *.html files from *.adoc
-    gradle -b $GRADLE_BUILD_FILE asciidoctor --info
```

Comment:
we are removing the gradle call entirely here, so why do we the need the `build.gradle`?

Reply:
Ah, now I see, it's still used for another task.

Reply:
@dhiller we still need gradle for the `swagger2markup` tool. I will try to get rid of this dependency as well, by installing it on the container, but I would like to do so in a separate PR, WDYT?

Reply:
@dhiller Sounds good to me. The reason I wanted to make this PR only for `asciidoctor` is because from tomorrow it won't be available anymore, therefore `make apidocs` will fail.

Reply:
@dhiller I meant to say that `asciidoctor` Gradle plugin is hosted only on one repo which is the Jcenter, and IIUC Jcenter will shut down since tomorrow.

Reply:
Sounds good to me to do this in a separate PR and tackle the urgent thing first.